api/client/samples/at-time-query-examples.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>"If a source&nbsp;is reporting a "total exports" number for the January 1st to December 31st..."</p><p> </p><p>The "the" before January is not needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
removed